setup.cfg

```diff
@@ -27,3 +27,8 @@ disallow_any = unannotated
 
 [mypy-tests.*]
 ignore_errors = true
+
+[pycodestyle]
+max-line-length = 160
+statistics = True
+
```

Comment:
Maybe this line can be removed. Last character in every file should be \n and not \n\n.